packages/plugin-mcp/src/collections/createApiKeysCollection.ts

```diff
@@ -147,7 +148,10 @@ export const createAPIKeysCollection = (
         admin: {
           description: 'The user that the API key is associated with.',
         },
-        relationTo: 'users',
+        relationTo:
+          typeof pluginOptions.userCollection === 'string'
+            ? pluginOptions.userCollection
+            : pluginOptions.userCollection?.slug || 'users',
```

Comment:
This could be improved. Since `pluginOptions.userCollection` is an optional field I am guessing most users won't have it configured. I would want the same fallback handling as we do for the admin user sanitization: https://github.com/payloadcms/payload/blob/91e64b3937765dfcf75011ae34f3c8710b0c9e81/packages/payload/src/config/sanitize.ts#L57

Always need to go for lowest amount of config possible and sensibly defaulting as we can.

Reply:
`userCollection` is now optional.